tuning/config/configs.py

```diff
@@ -0,0 +1,42 @@
+from dataclasses import dataclass, field
+from typing import Dict, Optional
+import transformers
+
+DEFAULT_CONTEXT_LENGTH=4096
+DEFAULT_OPTIMIZER="adamw_torch"
+
+IGNORE_INDEX = -100
+DEFAULT_PAD_TOKEN = "[PAD]"
```

Comment:
is this a valid token for the tokenizer used by llama? if not maybe we should use the unk token?